staging/src/k8s.io/cri-api/pkg/apis/testing/fake_runtime_service.go

```diff
@@ -418,7 +416,7 @@ func (r *FakeRuntimeService) StopContainer(containerID string, timeout int64) er
 
 	c, ok := r.Containers[containerID]
 	if !ok {
-		return fmt.Errorf("container %q not found", containerID)
+		return nil
```

Comment:
The fact that we return a container not found error here (and presumably test against this) makes me concerned that this behaviour was deliberate for some reason. Also, our tests and code _should_ be resilient to the container runtime returning an error here.